frontend/app/src/composables/message-handling.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable max-lines */
```

Comment:
We should probably figure out how to split the file further here. not now but we should start thinking about it